x11-misc/libinput-gestures/libinput-gestures-2.30.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6} )
+
+inherit python-single-r1
+
+DESCRIPTION="Actions gestures on your touchpad using libinput"
+HOMEPAGE="https://github.com/bulletmark/${PN}"
+SRC_URI="https://github.com/bulletmark/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk kde"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	>=dev-libs/libinput-1.8.0
+	x11-misc/xdotool
+	x11-misc/wmctrl"
+DEPEND=">=dev-libs/libinput-1.8.0
+	dev-util/desktop-file-utils"
+
+pkg_postinst() {
+	elog "You must be in the input group to read the touchpad device."
+
+	if use gtk ; then
+		if ! has_version x11-libs/gtk+:3 ; then
+			elog "You should install x11-libs/gtk+:3
```

Comment:
I think these messages should be the other way round:

"${PN}-setup script supports Gnome 3 via x11-libs/gtk+:3."

Reply:
Used your message.